data/Delimiter/bash_single_quote_escape.yaml

```diff
@@ -0,0 +1,4 @@
+type: string
+start: "'"
+end: "'"
+escape: "'\\'"
```

Comment:
I doubt that this is correct.

Please check the rendered version of this delimiter to see what it looks like after it has been parsed by yaml and then processed by statik

Reply:
I was a bit confused about this.

I googled "bash single quote escape" and this is what I found: https://stackoverflow.com/questions/1250079/how-to-escape-single-quotes-within-single-quoted-strings
So what I get is, in Bash, if you want to escape single quotes you have to put `'\'` before the single quote. Is there a better way? Or should Bash just use `single_quote.yaml` (assuming we can't escape)?